packages/block-library/src/post-author-biography/index.php

```diff
@@ -15,10 +15,11 @@
  */
 function render_block_core_post_author_biography( $attributes, $content, $block ) {
 	if ( ! isset( $block->context['postId'] ) ) {
-		return '';
+		$author_id = get_query_var( 'author' );
+	} else {
+		$author_id = get_post_field( 'post_author', $block->context['postId'] );
 	}
```

Comment:
Nit: Now this conditional is more than a simple guard clause, perhaps it should test for the positive scenario? Sort of like the code style guidelines recommend for [ternary operators](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#ternary-operator). 

I don't feel strongly about this but it might be good to have all three locations logic match at least 🤷

Reply:
Thanks for the review!

I rewrote the two places by [8151386](https://github.com/WordPress/gutenberg/pull/55451/commits/81513868e918c6c9865f668a63d44c98ee3265d1). If I rewrite [this part](https://github.com/WordPress/gutenberg/pull/55451/files#diff-4221bdc24052b61ece34522bef8a97b93315c029841b6dbdf44ac6d301a999c5R34-R40) using the ternary operator, nested conditional statements will occur, so I think it's fine as is, but what do you think?

Reply:
Apologies if I wasn't clear with my suggestion. I didn't mean that you needed to change these if/else statements to ternaries, just that we should check the positive case first. Like what is suggested for when you do use ternary operators. 

This is such a minor detail, don't feel you need to keep changing the code over it.

Below is a quick diff to illustrate what I was originally suggesting:

<details>
<summary>Example diff</summary>

```diff
diff --git a/packages/block-library/src/post-author-biography/index.php b/packages/block-library/src/post-author-biography/index.php
index 6c4cdc301e..23f5f16cbc 100644
--- a/packages/block-library/src/post-author-biography/index.php
+++ b/packages/block-library/src/post-author-biography/index.php
@@ -14,7 +14,11 @@
  * @return string Returns the rendered post author biography block.
  */
 function render_block_core_post_author_biography( $attributes, $content, $block ) {
-	$author_id = isset( $block->context['postId'] ) ? get_post_field( 'post_author', $block->context['postId'] ) : get_query_var( 'author' );
+	if ( isset( $block->context['postId'] ) ) {
+		$author_id = get_post_field( 'post_author', $block->context['postId'] );
+	} else {
+		$author_id = get_query_var( 'author' );
+	}
 
 	if ( empty( $author_id ) ) {
 		return '';
diff --git a/packages/block-library/src/post-author-name/index.php b/packages/block-library/src/post-author-name/index.php
index 92d37f70f9..a5fadfcbf3 100644
--- a/packages/block-library/src/post-author-name/index.php
+++ b/packages/block-library/src/post-author-name/index.php
@@ -14,7 +14,11 @@
  * @return string Returns the rendered post author name block.
  */
 function render_block_core_post_author_name( $attributes, $content, $block ) {
-	$author_id = isset( $block->context['postId'] ) ? get_post_field( 'post_author', $block->context['postId'] ) : get_query_var( 'author' );
+	if ( isset( $block->context['postId'] ) ) {
+		$author_id = get_post_field( 'post_author', $block->context['postId'] );
+	} else {
+		$author_id = get_query_var( 'author' );
+	}
 
 	if ( empty( $author_id ) ) {
 		return '';

```

</details>

Reply:
Ah, I mistakenly thought that I should use ternary operators itself. I understand what you mean 😅